.gitignore

```diff
@@ -0,0 +1,5 @@
+#ignore test.md file
+test.md
+
+#ignore MacOs default file
+.DS_Store
```

Comment:
- [ ] Kindly add `node_modules` to the `.gitignore` file to make your repo free from unwanted files.